presto-main-base/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -303,6 +303,7 @@ public class FeaturesConfig
     private boolean nativeExecutionTypeRewriteEnabled;
     private String expressionOptimizerName = DEFAULT_EXPRESSION_OPTIMIZER_NAME;
     private boolean addExchangeBelowPartialAggregationOverGroupId;
+    private boolean addDistinctBelowSemiJoinBuild;
```

Comment:
This can be on by default?

Reply:
I'd like to run it for some time before turning it on as default.